pkg/user/password_rest.go

```diff
@@ -0,0 +1,38 @@
+package user
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/oklog/ulid"
+)
+
+type PasswordReset struct {
+	Token     string
+	CreatedAt time.Time
+	ExpiresAt time.Time
```

Comment:
No need because just adding 24 hours to CreatedAt is enough to detect it is expired
```suggestion
```